packages/wrangler/src/config-cache.ts

```diff
@@ -0,0 +1,44 @@
+import { mkdirSync, readFileSync, writeFileSync } from "fs";
+import { dirname, join } from "path";
+import { cwd } from "process";
+
+interface ConfigCache {
+  account_id?: string;
+  pages_project_name?: string;
+}
+
+const relativeCacheLocation = join(
+  "node_modules",
+  ".cache",
+  "wrangler",
```

Comment:
let's make this wrangler-pages for now, we can't be sure whether folks use the same across their codebase. and this "name" should probably be the first arg in saveToConfigCache/getConfigCache

Reply:
If we are making this a generic module then it should also be that `getConfigCache()` and `saveToConfigCache()` are generic:

```ts
export const getConfigCache = <T>(cacheName: string): T { ... };
export const saveToConfigCache = <T>(cacheName: string, newValues: <T>): void => { ... };
```

Reply:
I'll make it a parameter and use `node_modules/.cache/wrangler/pages-config-cache.json` for Pages stuff. I figure if a `.cache` folder is shared between all `node_modules`, it would be polite to scope it by package name first.